lib/score.js

```diff
@@ -0,0 +1,24 @@
+class Score {
```

Comment:
I debated making this a separate class too (I did for Ruby, but not JS!) - I think it probably works better as a separate class